src/legacy/ui/field_formats/mixin/field_formats_service.test.ts

```diff
@@ -37,7 +37,7 @@ describe('FieldFormatsService', () => {
   let fieldFormatsService: FieldFormatsService;
 
   beforeEach(() => {
-    const fieldFormatClasses = [(createNumberFormat() as unknown) as FieldFormat];
+    const fieldFormatClasses = [(NumberFormat as unknown) as FieldFormat];
```

Comment:
It should work without casting. Could you please check it

const fieldFormatClasses = [NumberFormat];

Reply:
It would be great to get proposal how to get it more clearer. I've checked it twice and we aren't able to pass just `NumberFormat` or `NumberFormat as FieldFormat `

Reply:
Looks like you you made a mistake in FieldFormatsService. I see you expect in fieldFormatClasses argument a instance of FieldFormat but pass constructor function. As a result you see that issue. 

Please use  `const fieldFormatClasses = [NumberFormat]; `here

Reply:
Nice catch! Changed. Thanks